releasenotes/notes/tls-fc.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: networking
+issue:
+- 29538
+releaseNotes:
+- |
+  **Fixed** an issue causing one simple TLS requests from failing when permissive mode is enabled. This does not impact mTLS.
```

Comment:
Not quite correct grammatically. causing and failing probably should not both be -ing verbs. Also one requests should be one request or a request.

Reply:
"causing client side application TLS requests sent to PERMISSIVE mode enabled server fail."

Just feel simple TLS is a bit vague.

Reply:
updated